keps/sig-node/20180906-quotas-for-ephemeral-storage.md

```diff
@@ -622,15 +625,21 @@ appropriate end to end tests.
 The following criteria applies to
 `LocalStorageCapacityIsolationFSMonitoring`:
 
-Alpha
+## Phase 1: Alpha (1.15)
 
 - Support integrated in kubelet
+- Alpha-level documentation
+- Feature gate disabled by default
```

Comment:
The feature gate is probably not worth mentioning here or below.  It is a result of, not a requirement for graduation.